README.md

```diff
@@ -1,16 +1,164 @@
+manipulation of the user data.
+
+A Hazelcast cluster consists of one or more cluster members. These members generally run on multiple virtual or
+physical machines and are connected to each other via the network. Any data put on the cluster is partitioned to
+multiple members transparent to the user. It is therefore very easy to scale the system by adding new members as
+the data grows. Hazelcast cluster also offers resilience. Should any hardware or software problem causes a crash
+to any member, the data on that member is recovered from backups and the cluster continues to operate without any
+downtime.
+
+The quickest way to start a single member cluster for development purposes is to use our
+[Docker images](https://hub.docker.com/r/hazelcast/hazelcast/).
+
+```bash
+docker run -p 5701:5701 hazelcast/hazelcast
+```
+
+This command fetches the latest Hazelcast version. You can find all available tags
+[here](https://hub.docker.com/r/hazelcast/hazelcast/tags).
+
+You can also use our ZIP or TAR [distributions](https://hazelcast.com/open-source-projects/downloads/)
+as described [here](DOCUMENTATION.md#121-setting-up-a-hazelcast-cluster).
+
+### Client
+
+#### Installing
+
+##### Vcpkg Users
+Hazelcast C++ client package is available for [Vcpkg](https://github.com/microsoft/vcpkg) users. The port name is `hazelcast-cpp-client`.
+
+Please see [Getting Started](https://github.com/microsoft/vcpkg#getting-started) on how to use Vcpkg package manager with your application. In summary,
+
+If you use Linux or Mac:
+
+```sh
+git clone https://github.com/microsoft/vcpkg
+./vcpkg/bootstrap-vcpkg.sh
+./vcpkg/vcpkg install "hazelcast-cpp-client[openssl]" --recurse
+``` 
+
+If you use Windows:
+
+```bat
+git clone https://github.com/microsoft/vcpkg
+.\vcpkg\bootstrap-vcpkg.bat
+.\vcpkg\vcpkg install "hazelcast-cpp-client[openssl]:x64-windows" --recurse
+``` 
+The above code snippet will install `hazelcast-cpp-client` with its `boost` dependencies.
+
+After the installation, the library is available for usage. For example, if you are using CMake for your builds, you can use the following cmake build command with the `CMAKE_TOOLCHAIN_FILE` cmake option to be the `vcpkg.cmake`.
+```bat
+> cmake -B [build directory] -S . -DCMAKE_TOOLCHAIN_FILE=[path to vcpkg]/scripts/buildsystems/vcpkg.cmake
+> cmake --build [build directory]
+```
+
+The above code will install `hazelcast-cpp-client` with its `boost` and `openssl` dependencies.
+
+You can find more details on using a Vcpkg installed package from different IDEs in your projects from the [Vcpkg Official Getting Started](https://github.com/microsoft/vcpkg#getting-started) documentation.
+
+##### Other Methods
+
+You can also install the hazelcast-cpp-client with Conan and from source code. You can more information from [Reference Manual](https://github.com/akeles85/hazelcast-cpp-client/blob/readme_update/Reference_Manual.md#11-installing).
```

Comment:
`conan` might be a link to conan website. Some users might not familiar with it.

Reply:
Fixed